projects/dotcms-field-elements/src/components/dot-form/dot-form.e2e.ts

```diff
@@ -0,0 +1,106 @@
+import { newE2EPage, E2EPage, E2EElement } from '@stencil/core/testing';
+
+describe('dot-form', () => {
+    let page: E2EPage;
+    let element: E2EElement;
+    let formStatus = {};
+
+    const fields = [
+        {
+            fieldType: 'Text',
+            defaultValue: 'defaultValue1',
+            hint: 'hint1',
+            name: 'field1',
+            required: true,
+            value: 'value1',
+            variable: 'field1'
+        },
+        {
+            fieldType: 'Text',
+            defaultValue: 'defaultValue',
+            hint: 'hint2',
+            name: 'field2',
+            required: true,
+            value: 'value2',
+            variable: 'field2'
+        }
+    ];
+
+    beforeEach(async () => {
+        page = await newE2EPage();
+        await page.setContent(
+            `<dot-form submit-label="Save" reset-label="Reset"></dot-form>`
```

Comment:
`submit-label="Save" reset-label="Reset"` should be different than the fallback one and also need test to make sure they work